tasks/install.yaml

```
@@ -0,0 +1,30 @@
+- block:
+
+  - name: Add MariaDB repo key
+    apt_key:
+      url: https://mariadb.org/mariadb_release_signing_key.asc
+      state: present
+
+  - name: add mariadb repo
+    apt_repository:
+      repo: "deb http://{{ mariadb.mirror }}/mariadb/repo/{{ mariadb.version }}/ubuntu focal main"
+      state: present
+      filename: /etc/apt/sources.list.d/mariadb.list
+
+  when: ansible_distribution == 'Ubuntu'
+
+- name: Install MariaDB
+  when:
+    - ansible_distribution == 'Ubuntu'
+    - mariadb.galera.arbitrator == False
+  apt:
+    name: [mariadb-server, python3-pymysql, pwgen]
```

Multi-line arrays are better for Git blame :)
```suggestion
    name:
      - mariadb-server
      - python3-pymysql
      - pwgen
```